estate/models/estate_property.py

```diff
@@ -0,0 +1,99 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, models, fields
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
```

Comment:
The pep8 says that the external imports must be placed before the internal imports. You did it well but a blank line is expected between them.  

When you import several functions, classes from a same module, they must be sorted alphabetically.

```suggestion
from dateutil.relativedelta import relativedelta

from odoo import api, fields, models
from odoo.exceptions import UserError
from odoo.tools.float_utils import float_compare, float_is_zero

```